.github/workflows/pr-test.yaml

```diff
@@ -9,8 +9,9 @@ jobs:
   build-pipenv:
 
     runs-on: ubuntu-latest
-    strategy:
-      python-version: [3.7.1, 3.8, 3.9]
+    strategy:      
+      matrix:
+        python-version: [3.7.1, 3.8, 3.9]
```

Comment:
wondering why this is in this PR - seems like a separate concern?

Reply:
This was throwing an error while running tests. So this seemed to be a quick fix. not sure if it is supposed to be this way.